bin/run.sh

```diff
@@ -17,5 +17,32 @@ else
   fi
 fi
 
+
+if [[ -f VERSION ]]
+then
+  VERSION=$(cat VERSION)
+else
+  VERSION="Manual Deployment"
+fi
+
+NEW_RELIC_API_KEY=$(
+  echo "${VCAP_SERVICES}" | \
+  jq '.[] | map(select(.name == "tock-credentials-logging")) | .[].credentials.NEW_RELIC_API_KEY' -r
```

Comment:
##TODO

This needs to be updated to match `tock-credentials` which is discussed in `/docs/deployment-process.md`.